appveyor_native_deploy.yml

```
@@ -0,0 +1,18 @@
+clone_depth: 1
+version: '{build}'
+skip_non_tags: true
+image: Visual Studio 2017
+configuration: Release
+platform: Any CPU
+
+before_build:
+- cmd: nuget restore osu-framework-native/osu-framework-native.sln
+
+build_script:
+  - cmd: dotnet pack osu-framework-native/osu-framework-native /p:Version=0.0.%APPVEYOR_BUILD_VERSION%
```

```suggestion
  - cmd: dotnet pack osu-framework-native/osu-framework-native /p:Version=%APPVEYOR_BUILD_VERSION%
```

wrong build version. doesn't match deploy.